### ISSUE TITLE
Drawing multiple segmentation masks in a single binary image

### ISSUE THREAD

**Issue by guiaugustoga987** [ultralytics/ultralytics#5341]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello,

It might be a silly question, but I have a image with 2 objects and I need to draw both segmentation masks in a single binary image.

With the code : 

```
from ultralytics import YOLO
import cv2
import torch
import numpy as np

model = YOLO('path/to/model')
results = model('path/to/image', imgsz=640)

img = (results[0].cpu().masks.data[0].numpy() * 255).astype("uint8")

cv2.imshow("result", img)
cv2.waitKey(0)
```
I can access the mask with the highest probability. If I change to `img = (results[0].cpu().masks.data[1].numpy() * 255).astype("uint8") ` I can visualize the second mask. How do I merge both masks in a single binary image ?

Thanks.

**Comment by guiaugustoga987** [comment]
I found a solution. I used the cv2.add function as below : 

```
img = (results[0].cpu().masks.data[0].numpy() * 255).astype("uint8")

height,width = img.shape

masked = np.zeros((height, width), dtype="uint8")
num_masks = len(results[0].cpu().masks.data)
for i in range(num_masks):
    masked = cv2.add(masked,(results[0].cpu().masks.data[i].numpy() * 255).astype("uint8"))
```

**Comment by glenn-jocher** [comment]
@guiaugustoga987 thanks for joining the YOLOv8 community and for sharing your solution with us!

Your solution of using 'cv2.add()' function is a good way to merge multiple segmentation masks into a single binary image. Here, you are merging the masks by iterating over all available masks using 'cv2.add()', which carries out pixel-wise addition of two images of the same size and type.

Just a note, in your particular case, using 'cv2.add()' should be fine because you are working with binary masks (values are either 0 or 255). However, if the pixel values of overlapping regions in your masks were to exceed 255 when added together, remember that 'cv2.add()' would saturate them at 255 to prevent any overflow, maintaining the image validity.

I hope this information is helpful as you continue with your project. Please feel free to reach out if you have more questions or run into any other issues.
